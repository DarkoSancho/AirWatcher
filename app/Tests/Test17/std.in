1
Provider100
2
2