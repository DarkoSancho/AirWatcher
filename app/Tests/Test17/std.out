Welcome to AirWatcher App!

Menu:
1. Sign In
2. Exit
Select your choice: 
Enter user login: 

Menu:
1. Sign In
2. Exit
Select your choice: 
Exiting...
